homeassistant/components/light/__init__.py

```diff
@@ -302,6 +318,11 @@ def color_xy(self):
         return None
 
     @property
+    def color_ct(self):
```

Comment:
I'll change the name to color_temp (to keep it short).
And agreed, it should be used throughout hass